webapp/channels/src/components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -119,9 +119,6 @@ const ThreadList = ({
             }
         }
         select(data[threadIdToSelect]);
-
-        // hacky way to ensure the thread item loses focus.
-        ref.current?.focus();
```

Comment:
I'm definitely glad to get rid of this hack, but I see the post list scrolling when doing this which is why we did this hack before (https://github.com/mattermost/mattermost-webapp/pull/9122#issuecomment-945935497). Perhaps we could fix that by adding `e.preventDefault()` here?

Reply:
Didn't notice that before, adding `e.preventDefault` seems to work well here :)

However, the keyboard navigation still focuses the thread if there's a draft on the thread, so that can get a little wonky. Might not be a big deal, but worth calling out.

Reply:
Yeah, I feel like this UX is definitely not ideal between how the autofocus happens and the fact that the up and down arrows switch threads from the thread textbox. I'd expect that the normal workflow would be to tab onto the thread list, use the arrow keys to navigate to a thread, the user hits enter, and then their focus moves to the textbox. It's possible though that some users may be used to using the arrow keys from the textbox to switch threads though, so we'd need UX guidance if we were to make that change, and I think that's out of scope for these tickets anyway